org.eclipse.winery.repository.rest/src/main/java/org/eclipse/winery/repository/rest/resources/edmm/EdmmResource.java

```diff
@@ -87,6 +90,19 @@ public EdmmResource(TServiceTemplate element) {
         this.element = element;
     }
 
+    @GET
+    @Path("supportedTechnologies")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getPlugins() {
+        List<DeploymentTechnology> entity = PluginManager.getInstance().getPluginsList().stream()
+            .map(TransformationPlugin::getDeploymentTechnology)
+            .collect(Collectors.toList());
+        return Response.ok()
```

Comment:
Do you need the Response explicilty? Otherwise you can directly return the list and Jersy handels the rest on its own.